site/pages/api/type-of-consequence.test.ts

```diff
@@ -0,0 +1,103 @@
+import { describe, it, expect, afterEach, vi } from "vitest";
+import addConsequence from "./type-of-consequence.api";
+import { ConsequenceType, TransportMode } from "../../constants/enum";
+import {
+    COOKIES_ADD_CONSEQUENCE_INFO,
+    COOKIES_ADD_CONSEQUENCE_ERRORS,
+    ADD_CONSEQUENCE_PAGE_PATH,
+} from "../../constants/index";
+import { AddConsequenceProps, ErrorInfo } from "../../interfaces";
+import { getMockRequestAndResponse } from "../../testData/mockData";
+import * as apiUtils from "../../utils/apiUtils";
+
+const tenSeconds = 10000;
+
+describe("addConsequence", () => {
+    const writeHeadMock = vi.fn();
+    const setCookieSpy = vi.spyOn(apiUtils, "setCookieOnResponseObject");
+    afterEach(() => {
+        vi.resetAllMocks();
+    });
+
+    it("should redirect back to next page when all required inputs are passed", () => {
+        const disruptionData: AddConsequenceProps = {
+            modeOfTransport: TransportMode.bus,
+            consequenceType: ConsequenceType.operatorWide,
+        };
+
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Could we move these up to the top of the file and change them to whole file ignores? 

Like this:

`/* eslint-disable @typescript-eslint/no-unsafe-assignment */`
`/* eslint-disable @typescript-eslint/no-unsafe-argument */`

Reply:
done